features/cellular/framework/device/CellularStateMachine.cpp

```diff
@@ -373,6 +374,22 @@ void CellularStateMachine::state_device_ready()
             }
         }
     }
+
+#if MBED_CONF_MBED_TRACE_ENABLE
+    CellularInformation *info = _cellularDevice.open_information();
+    char buf[80]; // this should actually be 2048, but then you should use heap memory or likely run out of the stack
```

Comment:
This could be something like:
```
char *buf = new (std::nothrow) char[2048]; // size from 3GPP TS 27.007
if (buf) {
  if (info->get_manufacturer..
  ...
  delete[] buf;
}
```

Reply:
done